### ISSUE TITLE
About TopDownAffineTransform and padding

### ISSUE THREAD

**Issue by MiraPurkrabek** [open-mmlab/mmpose#1949]
Hello,

I'm trying to understand the data processing you're doing and and I'm a but confused regarding the pad for the TopDown approach. You mention in the #1686 that you pad the bounding box by the factor of 1.25. Where does this number come from?

If I understand correctly (as you hinted in #617), it is from AlphaPose and their Part-Guided Proposal Generator (PGPG) where they offset the bounding box RANDOMLY from a uniform distribution. Is your implementation just subsiding the random pad with a fixed one? Why so?

Second, the padding in PGPG means enlarging the bounding box (BB) and taking slightly larger part of the image. The zero-padding happenes only when the BB is so big you cannot take bigger part of the image (like when the whole picture is in the BB), right?

Third, you pad images for both train and test. According to the AlphaPose paper, the PGPG is employed only for the training phase to bridge the domain gap between detected BBs And ground-truth BBs. Can you please comment on that?

Any link or redirection to more literature would be also helpful.

Thank you for your help and this great repo.

**Comment by ly015** [comment]
Hi, thanks for using MMPose. The 1.25x bbox padding has been widely adopted by previous works like [SimpleBaseline](https://github.com/microsoft/human-pose-estimation.pytorch/blob/master/lib/dataset/coco.py) and [HRNet](https://github.com/HRNet/HRNet-Human-Pose-Estimation/blob/master/lib/dataset/coco.py). We follow this practice in MMPose. The padding is meant to enlarge the bbox area to include more contextual information for pose estimation. It is not directly related to PGPG. And zero-padding will only happen on the (enlarged) bbox area that is outside the image.

**Comment by MiraPurkrabek** [comment]
Hi, thank you for your quick response. I see the 1.25 scale in the linked code but did not find any mention of it in any of the papers. I got the intuitive explanation but do you know if someone justified this tweak in a ablation study?

**Comment by ly015** [comment]
I think 1.25 is just an empirical value that works fine on COCO. I am not sure if there is any extensive study on the padding ratio.
